src/invoices/InvoicesList/Invoice/InvoiceLineDetailsLayer.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { LoadingPane } from '../../../common/components';
+import {
+  invoiceResource,
+} from '../../../common/resources';
+
+class InvoiceLineDetailsLayer extends Component {
+  static manifest = Object.freeze({
+    invoice: invoiceResource,
+    query: {},
```

Comment:
is it used?

Reply:
or will be?

Reply:
it will be used in dropdown menu for edit invoice line at least